nmosauth/auth_server/security_api.py

```diff
@@ -122,14 +128,38 @@ def nameindex(self):
 
     @route(AUTH_VERSION_ROOT)
     def versionindex(self):
-        obj = ["register_client/", "revoke/", "authorize/", "token/", "certs/"]
+        obj = [
+            REGISTER_ENDPOINT + "/",
+            AUTHORIZATION_ENDPOINT + "/",
+            JWK_ENDPOINT + "/",
+            REVOCATION_ENDPOINT,
```

Comment:
Do these last two not need a trailing slash adding?

Reply:
Well... they dont exist with a leading slash as they're POST only endpoints.. so seemed silly to me to have hyperlinks leading to 404 pages. Makes it look like there's something wrong with the implementation.

Reply:
While true, this does make it diverge from all of the other specs.

Reply:
IS-04 Registration API /resource endpoint is used directly only for POST and is listed as `"resource/"` in the 'child resources response' from the base. To be fair it does have its own child resources for DELETE. I can't think of another example where there's a POST-only endpoint with a parent that returns a child resources response.